.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,60 @@
+name: Weekly maintenance/update tasks
+on:
+    # workflow_dispatch allows for manual triggering of the workflow
+    # Useful for testing that it works without waiting on the cron schedule.
+    workflow_dispatch:
+    schedule:
+        # https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
+        #        ┌─────────── minute (0 - 59)
+        #        │ ┌───────── hour (0 - 23)
+        #        │ │ ┌─────── day of the month (1 - 31)
+        #        │ │ │ ┌───── month (1 - 12 or JAN-DEC)
+        #        │ │ │ │ ┌─── day of the week (0 - 6 or SUN-SAT)
+        #        v v v v v
+        - cron: 30 7 * * 1
+        #       Run on Mondays @ 7:30am UTC (3:30am EDT, 12:30 am PDT)
+
+jobs:
+    update-browserslist:
+        name: Update the browserslist database (used by caniuse-lite)
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                os: [ubuntu-latest]
+                node-version: [20.x]
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Install & cache node_modules
+              uses: ./.github/actions/shared-node-cache
+              with:
+                  node-version: ${{ matrix.node-version }}
+                  ssh-private-key: ${{ secrets.KHAN_ACTIONS_BOT_SSH_PRIVATE_KEY }}
+
+            - name: Update browserslist DB
+              run: npx update-browserslist@latest
+
+            - name: Create Pull Request
+              uses: peter-evans/create-pull-request@v6
```

Comment:
suggestion: Would using `gh` be better than relying on a third-party action?

Also, I think enabling auto-merge for this PR would be a good idea. That way, when all actions are run AND the PR is reviewed, it will just land itself. What do you think?

Reply:
I'm going to stick with this action because it has some really useful behaviour that would take some amount of new code to reproduce: https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-behaviour

Also, auto-merge is not enabled on the Perseus repo and I'm a bit hesitant to open that can of worms. It's unclear how that would affect our OLC tooling and other workflows/tools we have in place.